### ISSUE TITLE
Needs mobile support

### ISSUE THREAD

**Issue by krschmidt** [LaneCommunityCollege/megamenu#1]
Currently set to disappear below 740px

**Comment by krschmidt** [comment]
By increasing support to 720px in https://github.com/LaneCommunityCollege/megamenu/commit/cd437fd6eecc25542ddf4099a8270bdff2a36a7c it's now possible for a site to move to mobile size, but the megamenu doesn't cooperate. Maybe add an option to include a variable for 'mobileView: 680px' and at sizes below that for now it disappears, but in the future it serves as an indicator for when to move to mobile (in which case mobile styles would need to apply based on screen size or class)?
![oops](https://cloud.githubusercontent.com/assets/605794/11643645/4835f270-9cfa-11e5-8788-f5f78fa28397.png)

**Comment by krschmidt** [comment]
One problem has always been the lack of a good way to point to open the menu, since the 5 categories won't be there. But, if we were to drop the logo from the megamenu, and make it a hamhurger menu, then add a menu option for Lane home, that'd create a pathway for doing a mobile menu.

**Comment by krschmidt** [comment]
@mandersj if this gets put on a website that uses a hamburger menu already, will it be problematic to have dueling burgers?

**Comment by mandersj** [comment]
Yeah... I think dueling burger is better but not by much. The LaneCC foundation using a little gift box. What about just using a "home" icon?

**Comment by krschmidt** [comment]
Home button would work to swap in for a hamburger right now, but not on mobile, where home would suddenly be everything.

**Comment by mandersj** [comment]
Right!... Dueling burgers it is. At least the burger indicates "menu" more than dueling logos.

**Comment by krschmidt** [comment]
To support a swipe to close: http://www.javascriptkit.com/javatutors/touchevents2.shtml
